code/src/App.js

```diff
@@ -1,9 +1,43 @@
-import React from 'react';
+import React, { useState } from 'react';
+import './button.css'
+
+import { StartSurvey } from './components/StartSurvey';
+import { Survey } from './components/Survey';
+import { Summary } from './components/Summary';
 
 export const App = () => {
+  const [section, setSection] = useState(0);
+  const [startDay, setStartDay] = useState('');
+  const [relationships, setRelationships] = useState('');
+  const [whatKinde, setWhatKinde] = useState('select below');
+  const [feeling, setFeeling] = useState('');
+
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
-  );
+    <main className="main">
+      <StartSurvey
+        setSection={setSection}
+        section={section} />
+
```

Comment:
The way you did the App.js is different from Daniel and instead you did the if conditional on the components. It's another cool way to do it. I learnt something new!